testing/icmake/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Shiz <hi@shiz.me>
+# Maintainer: Shiz <hi@shiz.me>
+pkgname=icmake
+pkgver=9.02.02
+pkgrel=0
+pkgdesc="a make utility using a C-like grammar"
+url="https://fbb-git.github.io/icmake/"
+arch="all"
+license="GPL3+"
+makedepends="bash"
+subpackages="$pkgname-doc"
+source="icmake-$pkgver.tar.gz::https://github.com/fbb-git/icmake/archive/$pkgver.tar.gz"
+builddir="$srcdir/icmake-$pkgver/icmake"
+options="!check"
+
+prepare() {
+	cd "$builddir"
+	sed -i '/^#define DOCDOCDIR/d' INSTALL.im
+	echo '#define DOCDOCDIR "usr/share/doc/icmake"' >> INSTALL.im
+}
+
+build() {
+	cd "$builddir"
+	./icm_prepare /
+	./icm_bootstrap x
```

Comment:
According to the docs: `(x is a dummy argument required by ./icm_bootstrap to run)`.